authentication/auth.go

```diff
@@ -239,10 +240,15 @@ func launcherSchemaFromURL(url string) (launcherSchema surveys.LauncherSchema) {
 		panic(err)
 	}
 
+	cacheBust := ""
+	if !strings.Contains(url, "?") {
```

Comment:
just wondering what would happen if url already contained `?bust` arg would the  appended timestamp bust overwrite it?

Reply:
it it already contains a `?bust` then this if would be false and nothing would happen

Reply:
sorry didn't see the `!`